vector/src/main/java/im/vector/notifications/NotificationUtils.kt

```diff
@@ -184,10 +187,14 @@ object NotificationUtils {
         i.flags = Intent.FLAG_ACTIVITY_CLEAR_TOP or Intent.FLAG_ACTIVITY_SINGLE_TOP
         val pi = PendingIntent.getActivity(context, 0, i, 0)
 
+        val accentColor = ContextCompat.getColor(context, R.color.notification_accent_color)
+
         val builder = NotificationCompat.Builder(context, LISTENING_FOR_EVENTS_NOTIFICATION_CHANNEL_ID)
-                .setContentTitle(context.getString(R.string.riot_app_name))
-                .setContentText(context.getString(subTitleResId))
+                .setContentTitle(context.getString(subTitleResId))
+                .setCategory(NotificationCompat.CATEGORY_PROGRESS)
```

Comment:
what is the effect of adding this category ?

Reply:
I haven't seen any effect, but I found it relevant to set it :)